docs/source/training/socialisation/inasafe_concepts.rst

```diff
@@ -1,33 +1,58 @@
 .. _inasafe_concepts:
 
-Key concepts in disaster management planning with |project_name|
-================================================================
-In this section we explain the basic concepts of |project_name| and the merits
-of disaster management planning.
+ |project_name| Concepts
```

Comment:
Hi @Charlotte-Morgan this become warning when i tried to build your file.

It said:
/inasafe-doc/docs/source/training/socialisation/inasafe_concepts.rst:8: WARNING: Block quote ends without a blank line; unexpected unindent